docs/migration/migrate_6_0_index.asciidoc

```diff
@@ -0,0 +1,338 @@
+[[migrating-6.0-index]]
+== Migrating Kibana index to 6.0
+
+[IMPORTANT]
+==============================================
+The following will make transformations to your Kibana index. It is recommended you backup your Kibana index prior to running any of the following commands.
+==============================================
+
+In order to migrate to Kibana 6, your Kibana index needs to be re-indexed. This will make transformations to the data model, removing types which are being {es-ref}/removal-of-types.html[removed in Elasticsearch].
+
+A UI is available in X-Pack 5.6 to assist with the migration. The following is provided should you be required to run the migration manually. The commands assume your using the default `.kibana` index. If `kibana.index` is set to something different in your `kibana.yml` you will need to modify the commands.
+
+To perform this migration without downtime, you will need to be running Kibana 5.6. This is the only version which supports both of the index types.
+
+. Set `.kibana` index to read-only:
++
+--
+[source,js]
+--------------------------------------------------
+PUT .kibana/_settings
+{
+  "index.blocks.write": true
+}
+--------------------------------------------------
+// CONSOLE
+--
+
+. Create `.kibana-6` index
++
+--
+[source,js]
+--------------------------------------------------
+PUT .kibana-6
+{
+  "settings" : {
+    "number_of_shards" : 1,
+    "index.format": 6,
+    "index.refresh_interval": -1
```

Comment:
we  probably want to turn this back on after, right?

Reply:
Actually, I am going to remove it. @archanid, is the upgrade UI turning this back on after the migration?